back_end/RayTracer/Models/Cameras/Camera.cs

```diff
@@ -5,25 +5,34 @@
 
```

Comment:
To be able to create an instance of camera, I changed the class from abstract to a normal class